gammapy/background/models.py

```diff
@@ -71,3 +77,491 @@ def evaluate(self, x, y):
         y = np.asanyarray(y, dtype=float)
         parvals = self.parvals(x)
         return self._evaluate_y(y, parvals)
+
+
+def _make_bin_edges_array(lo, hi):
+    """Make bin edges array from a low values and a high values array.
+
+    TODO: move this function to somewhere else? (i.e. utils?)
+
+    Parameters
+    ----------
+    lo : `~numpy.array`
+    	lower boundaries
+    hi : `~numpy.array`
+    	higher boundaries
+
+    Returns
+    -------
+    bin_edges : `~numpy.array`
+    	array of bin edges as [[low], [high]]
+    """
+    return np.append(lo.flatten(), hi.flatten()[-1:])
+
+
+class CubeBackgroundModel(object):
+    """Cube background model.
+
+    TODO: this is a prototype that can only read and plot a cube.
+
+    Parameters
+    ----------
+    det_bins : `~astropy.coordinates.Angle`
+        Spatial bin edges vector (low and high). X, Y are equivalent.
+    energy_bins : `~astropy.units.Quantity`
+        Energy bin edges vector (low and high).
+    background : `~astropy.units.Quantity`
+    	Background cube in (energy, X, Y) format.
+    """
+
+    def __init__(self, det_bins, energy_bins, background):
+        self.det_bins = det_bins
+        self.energy_bins = energy_bins
+
+        # TODO: what's the axes order?
```

Comment:
This is a tricky one ... can you resolve this TODO? How can we check if we get the X/Y axis order right?